### ISSUE TITLE
ranger magic moaning

### ISSUE THREAD

**Issue by angband-import** [angband/angband#3783]
**Reported by hboetes on 6 Apr 2013 22:12 UTC**
You hear something moaning in the distance.

It tries to cast a spell. It fails. You see an unhappy ranger. The ranger keeps nagging about not being able to cast spells.

So... I have 4 spellbooks with me and I can't cast any offensive spells accept for magic missile. OK, so being a ranger you aren't supposed to be able to use all those crafty spells that work for a sorcerer...

OK... But I still have to drag around 4 magic books. Isn't it time to have a new special edition for rangers which has the following spells:

trap/door destruction
cure poison
teleport self
satisfy hunger
turn stone to mud
reveal monsters
identify

in other words all the non offensive spells from b & c? Because the rest of the spells I can simply forget about. They cost way to much mana even if they succeed to be useful.

**Comment by angband-import** [comment]
**Comment by takkaria on 8 Apr 2013 00:23 UTC**
This is a major gameplay suggestion so I'd suggest you talk about it on the forums where it can be discussed with other players.

**Comment by angband-import** [comment]
**Comment by NickMcConnell on 7 Feb 2015 10:26 UTC**
Examine with new classes in 4.2

**Comment by angband-import** [comment]
**Comment by NickMcConnell on 15 Sep 2018 21:17 UTC**
New comment - their archery is nerfed too much currently

**Comment by angband-import** [comment]
**Comment by NickMcConnell on 29 May 2019 20:54 UTC**
Unnerfed probably enough in 155ae9ebdc
